### ISSUE TITLE
VAMC/Left Nav IA Feedback

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-cms#13908]

**Comment by mmiddaugh** [comment]
Moving this from #13895 so we can have everything in one place. 

The Left Navigation menu was not intended to appear on VAMC pages deeper than L3. The spoke label "ABOUT VA PITTSBURGH" is counted as 1 - even though it's not clickable.

However, the Left Nav appears on [example level 4 page](https://www.va.gov/pittsburgh-health-care/work-with-us/internships-fellowships/psychology-internships-and-fellowships/).

This was based on a belief that the Left Nav could not indent to this level. this page proves it is possible so we need to determine the intent given this new information.

[Link to Slack thread](https://dsva.slack.com/archives/C0FQSS30V/p1605737661089700?thread_ts=1605733478.079900&cid=C0FQSS30V)

<details>
<summary> Screenshot </summary>

![image.png](https://images.zenhubusercontent.com/5d8a47393ac3440001d1a2ed/d5bcc2b8-4039-46a9-9d9c-6614318a9981)
</details>

**Comment by davidmpickett** [comment]
Curious about "The Left Navigation menu was not intended to appear on VAMC pages deeper than L3" as it may relate to Police Data pages. Is the idea that L4 pages have would no Left Nav? Because if the police data page per facility is nested under the facility, then the police data pages will be L4

**Comment by mnorthuis** [comment]
> The Left Navigation menu was not intended to appear on VAMC pages deeper than L3. The spoke label "ABOUT VA PITTSBURGH" is counted as 1 - even though it's not clickable.

@mmiddaugh I'm curious where this came from?  I was under the impression that one of the biggest reasons for this version of the left nav was so it would always show the full nav all the way down to the level you are at - which is something thing the benefits left nav does not, it resets instead of showing the full nav.  

In addition, within the last year-ish, there were changes made to the VAMC sites to not allow nesting of pages beyond level 4 because we knew the left nav didn't indent that far...although I think it tried to :-) 

Also want to clarify how you identify the level of a page - the spoke is not a level as it is not clickable and not a part of the structure.  You can use the segments in the URL or the breadcrumb to count the levels.  VA.gov home is always 0 (root level). 

@davidmpickett regarding the police pages - is there intended to be a police page for every location or just a page at the system level for all locations?  This was something that was not clear to us.  The request came through looking like it was 1 page per system.

**Comment by rtwell** [comment]
@mnorthuis you are correct—the limit to the number if levels was put in place to prevent too much nesting (as the legacy IA had in spades). Given that structure, the full nav can and would be persistently displayed so users didn't have to fumble around with opening and closing.

**Comment by davidmpickett** [comment]
> @davidmpickett regarding the police pages - is there intended to be a police page for every location or just a page at the system level for all locations? This was something that was not clear to us. The request came through looking like it was 1 page per system.

@mnorthuis Apologies for any confusion. We're still nailing down the content model for the police pages. Current thinking is that there will be both:
- A system-level page with general information about VA police, system-level contact info, & links to data pages per facility
- A data page per facility 

Hopefully by our kick-off meeting next week we'll have more certainty and updated designs to reflect this

**Comment by mnorthuis** [comment]
@davidmpickett thank you - just noting here that I don't believe the individual location pages currently have child pages below them.  At one time there was the concept of having a facility map under each location, and the potential to have a mobile unit schedule, but I'm not sure where those landed. So, if we have never placed a child page under an individual location, there may be some additional considerations needed.

**Comment by Agile6MSkinner** [comment]
@mmiddaugh The original ticket is over four years old, so I'm not sure how much of what was referenced is still reflected in the links. I'm not seeing the issues identified but I'm not an expert on sidenav and breakcrumb details. Is this still something we need to be tracking?

**Comment by mnorthuis** [comment]
Status of issues referenced in this ticket:

> 1. For some L3 content that is hidden in the left nav, the left nav is still displayed on the page but the left nav no longer has a "you are here" indicator. Examples:
> https://staging.va.gov/pittsburgh-health-care/women-veteran-care/
> https://staging.va.gov/pittsburgh-health-care/internships-fellowships/forms/

**# 1 STATUS:** This is still an open issue. Recommend breaking this into its own ticket. 

> 2. URLs do not reflect that these are L3 pages - their breadcrumbs are accurate but URLs are not

**# 2 STATUS:** This has been fixed. 

> 3. The "Home" portion of the breadcrumb disappears on some pages, but not all. For example, it is not shown Health services and Locations pages, but does appear on Events and About us.
>     - "Home" should take you to va.gov home page.
>     - "Pittsburgh health care" should go back to the health care system primary page.

**# 3 STATUS:** The has been fixed.  The new breadcrumb displays "VA.gov home" as the first segment. 

> 4. The L2 options in the left nav do not all have hover states when they are expanded to show L3 items, but they do have hover states there are no L3s to show.
>     - Per @rtwell, when a user is on a page, that page in the side nav should not be clickable or have a hover state (because the user is on that page).

**# 4 STATUS:** This appears to have been fixed, however I'm curious if this meets today's standards?  Question for design system team and can be broken into separate issue if needed. 

> 5. For content that is at the top level of the VAMC site, and not part of a "spoke", there is no "You are here" indicator in the left nav. The left nav should be designed to accommodate this, or it should be removed, or the IA should be reviewed for these pages.
>     - Example: https://staging.va.gov/pittsburgh-health-care/make-an-appointment/

**# 5 STATUS:**  This is not fixed.  I recommend the IA of these pages is reviewed to determine if they should be shifted into one of the spokes and a proper place in the hierarchy, or if left nav changes need to be made to accommodate.  Recommend breaking this off into its own ticket. 

> 6.  For pages that are hidden in the left nav (e.g. all of the individual News releases):
>     - their respective breadcrumb would be VA Pittsburgh health care > News releases > News release title
>     - The side nav is hidden

**# 6 STATUS:** The breadcrumbs have been corrected, but the side nav should actually not be hidden.  This is related to #1 in this list and should be included in that issue. 

> 7. The Left Navigation menu was not intended to appear on VAMC pages deeper than L3. The spoke label "ABOUT VA PITTSBURGH" is counted as 1 - even though it's not clickable.  (From https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13895) 
>     - However, the Left Nav appears on [example level 4 page](https://www.va.gov/pittsburgh-health-care/work-with-us/internships-fellowships/psychology-internships-and-fellowships/).
>     - This was based on a belief that the Left Nav could not indent to this level. this page proves it is possible so we need to determine the intent given this new information.

**# 7 STATUS:** This was actually not an issue, the left nav should show every level.  We have, however, capped the hierarchy at 4 levels to ensure the nesting does not get out of hand and the left nav indenting is not as much of an issue.
